README.adoc

```diff
@@ -90,7 +90,7 @@ repositories {
 dependencies {
     ...
     compile("org.springframework.cloud:spring-cloud-gcp-starter-pubsub:1.0.0.M2")
-    compile("org.springframework.integration:spring-integration-core:5.0.1.RELEASE")
+    compile("org.springframework.integration:spring-integration-core:5.0.1")
```

Comment:
There is no such a version in Maven: https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.springframework.integration%22%20AND%20a%3A%22spring-integration-core%22.

What made you to remove `RELEASE` suffix ?

Reply:
The version seems to be in Maven Central, judging from the link you pasted.
The version is also omitted from the code. I tested and it works.

Reply:
Bear in mind the following:
`apply plugin: 'io.spring.dependency-management'`

Reply:
??? 
You are confusing me here.
And I guess you will confuse people who is going to download this Getting Started Guide.

So, what we had before was:
```
compile("org.springframework.integration:spring-integration-core:5.0.1.RELEASE")
```
And that's correct from our versions perspective.
What you have now is:
```
compile("org.springframework.integration:spring-integration-core:5.0.1")
```
And that's is wrong. Because we produce versions with the `RELEASE` suffix.

What you are going to tell that there is `io.spring.dependency-management' and it is good to manage versions for us automatically. Then this must be like:
```
compile("org.springframework.integration:spring-integration-core")
```
Fully without version.

What am I missing else, please ?

Reply:
Woops, my eyes detected something weird, but I couldn't tell what it was.
Thanks!